content/user/Introduction/managing-user-accounts.adoc

```diff
@@ -23,6 +25,89 @@ as Username, First Name, Last Name, Title etc.
 
```

Comment:
(GitHub doesn't let me add comments on other lines of the file, only on where changes were made in these cimmits) 
Please fix typo further below in this file, some titles read `Resetting a Users Preferences` and `Resetting a Users Homepage` and they should be possessives, `User's`